tests/regrtest_data/unicode/invisible_function.txt

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env python3
```

Comment:
Python/Pylint crashes with non braking space. To prevent future (or maybe if current, to be honest I don't remember) issues, if the folder is not properly ignored I named it .txt instead of *.py.

Reply:
Python crash with non-breaking spaces ? I don't think so, we had a lot of those caused by fat fingering them after comment until very recently (#5256). Or you're talking about something else than NBSP ?

Reply:
Sorry I meant zero-width-space.

Anyway the file does not work in Python at the moment (there is a unittest to verify that).
So as it is not a valid Python Script, I called it .txt.
See `test_finds_bidirectional_unicode_that_currently_not_parsed` for background on the file.

Reply:
All right, that make sense.